### ISSUE TITLE
Perform cross-browser testing of front-end graphics

### ISSUE THREAD

**Issue by mileswwatkins** [nprapps/elections18-graphics#6]
During development, we'll want to split up responsibilities so that several of these OS+browser combinations are tested at each test/demo.

---

[Browsers to test, per DM](https://confluence.npr.org/pages/viewpage.action?spaceKey=QA&title=Browser+and+OS+Support):

#### Windows 10

- Chrome (latest)
- Firefox (latest)

#### MacOS (latest)

- Chrome (latest)
- Firefox (latest)
- Safari (latest)

#### iOS (latest)

- Safari (latest)

#### Android (latest)

- Chrome (latest)

#### Windows 7

- Internet Explorer 11
- Edge (latest?)

**Comment by mileswwatkins** [comment]
[HDD template for functional testing](https://docs.google.com/document/d/1WA4ldMcSFfSAmp8UWgo1sVrwxcOc9fU8N6mbeaNjJIQ)

**Comment by mileswwatkins** [comment]
I checked most of these during today's test, using BrowserStack. Didn't notice anything major, besides one or two styling issues.

Would like to do another round of testing in a week or two to follow up. Specifically, would like to check:
- do links always open in a new tab when they should, and in the same window when they should?
- does the browser actually redraw numbers/colors/winners when new data comes in?

Otherwise, I think we're good. The visual style and components shouldn't change much between now and election day, and those all check out across browsers right now, save for a couple of issues I've ticketed.
